roles/monitored/tasks/main.yml

```diff
@@ -14,3 +14,11 @@
   import_tasks: raid.yml
   tags: nrpe_raid
   when: "'tags_raid' in group_names"
+- name: Configure temperature checks
+  import_tasks: temperature.yml
+  tags: nrpe_temperature
+  when: "ansible_facts['virtualization_role'] in ('NA', 'host')"  # physical machines
+- name: Configure conntrackd checks
+  import_tasks: conntrackd.yml
+  tags: nrpe_conntrackd
+  when: "gw_conntrackd_peer is defined"  # anything that has a conntrackd peer must be running it
```

Comment:
I wonder if this should check `'device_roles_router' in group_names`, to avoid relying on variables specific to the gw role outside of the role?

Reply:
That may be a better option.  I was just not aware if/how that could be done.  I will amend.

Reply:
Fixed this is [3314cf1](https://github.com/sown/ansible/pull/66/commits/3314cf19914146fa4a4bb0b8f81c55150ea49d19).